src/jitlayers.cpp

```diff
@@ -70,6 +69,7 @@ using namespace llvm;
 #include "julia_assert.h"
 
 RTDyldMemoryManager* createRTDyldMemoryManager(void);
+using namespace llvm::object;
```

Comment:
Without it I get a compilation error : `Owning Binary not declared in this scope`.

Reply:
Instead of using the namespace can you prefix the names?  If it is a lot more than a handful place the namespace is ok, but I suspect it is only in one or two places.

Reply:
Oh, yeah effectively it can be more convenient :D